.github/workflows/buildAndTestRyzenAI.yml

```diff
@@ -75,3 +75,4 @@ jobs:
 
             sed -i.bak 's/-sv --timeout 600/-sv --timeout 600 -j12 --show-unsupported/g' $PWD/../test/CMakeLists.txt
             ninja check-aie
+            ninja reference_designs
```

Comment:
`ninja check-reference-designs` ?

I've only ever run the reference designs with `lit` directly. I just tried via ninja and get unresolved tests and errors...

Reply:
> `ninja check-reference-designs` ?

uh whoops

> I've only ever run the reference designs with `lit` directly.

TIL! How do you run with lit directly?

Reply:
> TIL! How do you run with lit directly?

I do one of these:
```
$ pip install lit
$ cd <mlir-aie build dir>
$ lit [options] reference_deisgns/ipu-xrt
```
or
```
$ cd <mlir-aie build dir>
$ python <llvm build dir>/bin/llvm-lit [options] ...
```